README.md

```diff
@@ -1,84 +1,44 @@
+# Twilio Connector
 
-[Twilio](https://www.twilio.com/) is a cloud communications platform for building SMS, Voice & Messaging applications on 
-an API built for global scale.
-
-### Why would you use a Ballerina endpoint for Twilio
-
-Ballerina Twilio endpoint allows you to access the [Twilio REST API](https://www.twilio.com/docs/api) and perform 
-actions like sending a simple text message, making a voice call etc.
+Twilio connector provides a Ballerina API to access the [Twilio REST API](https://www.twilio.com/docs/api).
 
 ## Compatibility
 
-| Language Version           | Endpoint Version    | Twilio Basic API Version | Twilio Authy API Version |
-| -------------------------- | ------------------- | ------------------------ | ------------------------ |
-| 0.970.0-beta1-SNAPSHOT     | 0.5.10              | 2010-04-01               | v1                       |
-
-##### Prerequisites
-
-1. Download the ballerina [distribution](https://ballerinalang.org/downloads/).
+| Ballerina Language Version  | Twilio API Version | Twilio Basic API Version | Twilio Authy API Version |
+| :--------------------------:|:------------------:|:------------------------:|:------------------------:|
+| 0.970.0-beta2               | 0.8.0              | 2010-04-01               | v1                       |
 
-2. Create a Twilio account (https://www.twilio.com/) and obtain the following parameters:
-* ACCOUNT SID
-* AUTH TOKEN
+## Getting started
 
-3. Create a Twilio Authy app withing Twilio account (https://www.twilio.com/console/authy/applications) and obtain the
-following parameters:
-* PRODUCTION API KEY
+1.  Refer https://ballerina.io/learn/getting-started/ to download and install Ballerina.
+2.  To use Twilio endpoint, you need to provide the following:
 
-IMPORTANT: These tokens can be used to make API requests on your own account's behalf. Do not share these credentials.
+       - Account SId
+       - Auth Token
+       - Authy API Key
 
-### Getting started
+       *Please note that, providing Authy API Key is required only if you are going to use Authy related APIs*
 
-* Import the package to your ballerina project.
-```
-import wso2/twilio;
-```
-This will download the twilio artifacts from the central repository to your local repository.
+4. Create a new Ballerina project by executing the following command.
 
-### Working with Twilio Endpoint Actions
+      ``<PROJECT_ROOT_DIRECTORY>$ ballerina init``
 
-All the actions return `twilio:<Custom_Object>` or `twilio:TwilioError`. If the action was a success, then the 
-requested object will be returned while the `twilio:TwilioError` will be **empty** and vice-versa.
+5. Import the Twilio package to your Ballerina program as follows.
 
-##### Example
-* Request 
-```
+```ballerina
     import wso2/twilio;
 
-    public function main (string[] args) {
+    function main (string... args) {
         endpoint twilio:Client twilioClient {
              accountSid:config:getAsString(ACCOUNT_SID),
```

Comment:
Give an indication that ACCOUNT_SID needs to be changed.

Reply:
Fixed.